http-generator-client/src/test/java/io/avaje/http/generator/client/clients/Params.java

```diff
@@ -0,0 +1,49 @@
+package io.avaje.http.generator.client.clients;
+
+import io.avaje.http.api.Cookie;
+import io.avaje.http.api.Header;
+import io.avaje.http.api.QueryParam;
+
+public class Params {
```

Comment:
This does not add anything to the test coverage?

We have `@BeanParams` in tests/test-client-generation in JunkApi and CommonParams and if we needed more coverage we would add it there?

I'll remove this I think.

Reply:
Perhaps so, but these generator tests are what allow me to properly debug the processor with my IDE.